docs/source/additional_features.rst

```diff
@@ -138,7 +138,7 @@ Example
 
     constrained_optimizer = cooper.ConstrainedOptimizer(
         formulation=formulation,
-        primal_optimizer=primal_optimizer,
+        primal_optimizers=[primal_optimizer],
```

Comment:
Again, let's pass an optimizer object and not a list in the case of a single primal optimizer.

```suggestion
        primal_optimizers=primal_optimizer,
```